### DISCUSSION TITLE
Helligkeitseinstellung

### DISCUSSION THREAD

**Discussion by Hotzi57** [beabel/radiowecker#45] [Ideen / Ideas]
Ich hatte heute ein seltenes Phänomen. Da mein Radio noch in der Versuchsphase ist, habe ich es heute ausversehen ohne den Lichtsensor betreiben wollen. Er war nicht angeschlossen, aber die Helligkeit auf 0% eingestellt. Als ich das Radio an den Strom angeschlossen hatte, war alles finster. Lautsprecher war auch nicht dran.
Da bekam ich Panik, warum geht nichts!?
Es hat eine Weile gedauert, bis ich die Fälle gefunden habe.
Nun mein Gedanke, könnte man nicht über den Webserver auch die Helligkeit einstellen? Lautstärke geht ja auch schon.
Gruß, Andreas

**Comment by JeaneLG** [comment]
in den Fall einfach auf das Display tippen: Dann kommst du in den radio modus und zumindest bei mir dreht er da die Helligkeit hoch. Habe auch den Lichtsensor.

**Comment by fraeggle** [comment]
Falls jemandem die Helligkeit im Dunkeln noch zu Hell ist, oder man es im Dunkeln ganz ausschalten will:

Über Seiellen Monitor schauen, wass in diesem abgedunkeltem Zustand der ldr Wert ist. War bei mir z.B.12,
ging also nie ganz runter.....
Dann kann man folgende Zeile im tft_Display anpassen:
(ab einem ldr Wert von 20 wird dann bei mir das Display auf die geringst Mögliche Stufe gestellt)

  if (ledb < 20) ledb = 1;  //minimal brightness
Wenn es ganz aus sein soll:
  if (ledb < 20) ledb = 0;  //minimal brightness

Vielleicht hilft es ja jemandem.
